conversion-measurement/functions/apps/adtech.js

```diff
@@ -251,8 +251,8 @@ adtech.get('/conversion', (req, res) => {
 
   // Debug report (common to event-level and aggregate)
   console.log("Conversion Cookies Set: ", req.cookies)
-  // Optional: set a debug key, and give it the value of the legacy measurement 3P cookie.
-  // This is a simple approach for demo purposes. In a real system, you would still make this key a unique ID, but you may map it to additional trigger-time information that you deem useful for debugging or performance comparison.
+    // Optional: set a debug key, and give it the value of the legacy measurement 3P cookie.
+    // This is a simple approach for demo purposes. In a real system, you would still make this key a unique ID, but you may map it to additional trigger-time information that you deem useful for debugging or performance comparison.
```

Comment:
nit: Are these whitespace changes necessary?

Reply:
They are not, seems visual code introduced them, I can remove them.